samples/syelog/syelogd.cpp

```diff
@@ -180,9 +181,15 @@ PCLIENT CreatePipeConnection(HANDLE hCompletionPort)
         MyErrExit("CreateIoComplietionPort pClient");
     }
 
-    if (!ConnectNamedPipe(hPipe, pClient)) {
-        if (GetLastError() != ERROR_IO_PENDING &&
-            GetLastError() != ERROR_PIPE_LISTENING) {
+    if (!ConnectNamedPipe(hPipe, pClient)) {
+        DWORD dwLastErr = GetLastError();
+        //client already connected while call this fun

```

Comment:
Small nitpick, I think this comment could be more clear, something  like: 
```suggestion
        // Handle race between multiple client connections

```